### DISCUSSION TITLE
Repository moving to organization.

### DISCUSSION THREAD

**Discussion by bafolts** [bafolts/terraforming-mars#3689] [General]
This project has grown larger than I would have ever imagined when I first started this repository. There are now many contributors and players. With the project under my personal `bafolts` account there are various settings that only I have access to. This makes me a road block and a gate keeper. To allow more people to run the repository it is going to move to an organization. The plan is to move the repository to the [terraforming-mars](https://github.com/terraforming-mars) organization. This will give more access to more people for maintaining the project and integrating with other services.

The transition should be seamless and github should end up redirecting to the new location once it moves. @kberg and myself are doing trial runs to ensure once we do the move we don't lose any of the issues or git history. Unless there is any objection I will click the transfer button on Tuesday October 12th.

**Comment by kberg** [comment]
This also allows us to have a central place to create ancillary projects related to the server, but not necessarily part of the server. For instance, one that comes to mind is a UI that lets someone carefully curate a set of cards in a deck to upload to a JSON file. That's just a crazy idea. :D

**Comment by kberg** [comment]
I ran a test this morning and it worked as expected: github.com/kberg/test-repo redirects to github.com/terraforming-mars/test-repo. Both the open pull request and issue moved over too.

**Comment by kberg** [comment]
This is done.
